inc/metadata/namespace.php

```diff
@@ -8,6 +8,7 @@
 use function \Pressbooks\Utility\is_assoc;
 use function \Pressbooks\Utility\oxford_comma;
 use function \Pressbooks\Utility\oxford_comma_explode;
+use function Pressbooks\L10n\get_locale;
```

Comment:
New coding standards will likely nag us here. 

Use forward slash for functions, alphabetical order, functions first.